src/lib.rs

```diff
@@ -293,12 +294,67 @@ impl Frame {
         Ok(())
     }
 
+    /// Get the module's name by walking /proc/self/maps
+    #[cfg(all(
+        feature = "print-addresses",
+        unix,
+        not(any(target_os = "macos", target_os = "ios"))
+    ))]
+    fn module_info(&self) -> Result<(String, usize), String> {
+        use regex::Regex;
+        use std::path::Path;
+        let re = Regex::new(r"^(?P<start>[0-9a-f]{8,16})-(?P<end>[0-9a-f]{8,16}) (?P<perm>[-rwxp]{4}) (?P<offset>[0-9a-f]{8}) [0-9a-f]+:[0-9a-f]+ [0-9]+\s+(?P<path>.*)$")
```

Comment:
Please break this into multiple lines. I think you should be able to use the `(?x)` modifier for this, as shown in the second example code here:

https://docs.rs/regex/1.4.6/regex/#example-replacement-with-named-capture-groups